### ISSUE TITLE
Build: cmake won't generate ninja anymore

### ISSUE THREAD

**Issue by emanuele6** [SerenityOS/serenity#2423]
Since #2410 was merged, `cmake` fails to generate `build.ninja` files (`-GNinja`).

It is still able to generate `make` though.
```
CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/home/emanuele6/pro/serenityOS/serenity/Build' '-t' 'cleandead'

  failed with:

   ninja: error: build.ninja:153: multiple rules generate _disk_image [-w dupbuild=err]

  



CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/home/emanuele6/pro/serenityOS/serenity/Build' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:153: multiple rules generate _disk_image [-w dupbuild=err]

  



CMake Generate step failed.  Build files cannot be regenerated correctly.
```

**Comment by emanuele6** [comment]
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6294e7c99..dc45327bd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ add_custom_target(run
 
 add_custom_target(grub-image
     COMMAND ${CMAKE_COMMAND} -E env "SERENITY_ROOT=${CMAKE_SOURCE_DIR}" ${CMAKE_SOURCE_DIR}/Meta/build-image-grub.sh
-    BYPRODUCTS ${CMAKE_BINARY_DIR}/_disk_image
+    BYPRODUCTS ${CMAKE_BINARY_DIR}/_disk_image_
     USES_TERMINAL
 )
```
This change solves it: probably `ninja` can't handle two targets with the same byproduct.

**Comment by etaIneLp** [comment]
Fixed in #2424.
